src/App.jsx

```diff
@@ -1,9 +1,96 @@
 import './App.css'
+import Button from "./components/Button.jsx";
+import Bag1 from "./assets/bag_1.png";
+import Bag2 from "./assets/bag_2.png";
+import Bag3 from "./assets/bag_3.png";
+import Bag4 from "./assets/bag_4.png";
+import Product from "./components/Product.jsx";
+import Brand from "./assets/brand.png";
+import OurStory from "./assets/our_story.png";
+import Tile from "./components/Tile.jsx";
+
+function clickHandler(e) {
+    console.log(e.target);
+}
 
 function App() {
-  return (
-      <h1>Handbags & Purses</h1>
-  )
+    return (
+        <>
+            <h1>Handbags & Purses</h1>
+            <nav>
+                <Button
+                    text="to the collection"
+                    onClickHandle={clickHandler}
```

Comment:
Netheid: function in je button.jsx bouwen. Dan hoef je geen extra function hierin aan te roepen